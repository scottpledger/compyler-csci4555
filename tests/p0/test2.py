input()
print input()
print -------input()
input() + --input() + --------input()
print input() + --input() + --------input()
1
print 1
-1
print -1
x = 10
-x
print x
print -x
1 + 3
print 1 + 3
1 + -5
print 1 + -5
y = 1 + 3
print y
z = 1 + -5
print z
a = 1 + 2 + -3 + -7 + -8
print a
b = x + y + z + input() + -input() + a + -x + y
print b
print -----b
c = --x + -----y + --z + -7 + ------8 + --b
print c
print --x + -----y + --z + -7 + ------8 + --b
