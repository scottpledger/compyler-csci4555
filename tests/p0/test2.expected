1
-1
3
1
-1
10
-10
4
-4
4
-4
-15
-11
11
-8
-8
